src/website/src/pages/ec/components/page-banner/docs/usage.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  A page banner displays a prominent message and related optional actions.
```

Comment:
```suggestion
  A page banner displays a prominent message and related optional actions.
```
A page banner is smaller in size than a hero banner, and it is to be used for campaigns or on the homepage (where it replaces the page header component.
```